### ISSUE TITLE
CI - Increase timeout for Upgrade test

### ISSUE THREAD

**Issue by tjtelan** [infinyon/fluvio#2254]
[This PR ](https://github.com/infinyon/fluvio/pull/2252) took forever to merge because the upgrade test was being flaky

~Set env var `FLV_SOCKET_WAIT` to 5 minutes (300 seconds) to hopefully increase the stability of this test~

Set timeout on the step, https://github.com/infinyon/fluvio/issues/2254#issuecomment-1067430498

**Comment by sehz** [comment]
Actually, try to increase Github action timeout:
```
 - name: Run upgrade test with CI artifacts
        timeout-minutes: 5
```

to 10 minutes

**Comment by tjtelan** [comment]
Closed by #2253
